dlp/snippets/deid/deid_fpe.go

```diff
@@ -0,0 +1,101 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package deid
+
+// [START dlp_deidentify_fpe]
+import (
+	"context"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+
+	dlp "cloud.google.com/go/dlp/apiv2"
+	dlppb "google.golang.org/genproto/googleapis/privacy/dlp/v2"
+)
+
+// deidentifyFPE deidentifies the input with FPE (Format Preserving Encryption).
+// keyFileName is the file name with the KMS wrapped key and cryptoKeyName is the
+// full KMS key resource name used to wrap the key. surrogateInfoType is an
+// optional identifier needed for reidentification. surrogateInfoType can be any
+// value not found in your input.
+func deidentifyFPE(w io.Writer, client *dlp.Client, project, input string, infoTypes []string, keyFileName, cryptoKeyName, surrogateInfoType string) error {
```

Comment:
Include sample argument values.

https://github.com/GoogleCloudPlatform/golang-samples/blob/master/CONTRIBUTING.md#function-arguments

Reply:
I tried to copy arguments from the Python Node.JS samples on https://cloud.google.com/dlp/docs/deidentify-sensitive-data. Should I keep the explanations in the function comment?

Reply:
Yes, I think the explanations are fine to keep.